Author: Ole Streicher <olebole@debian.org>
Description: Use system provided libwcstools instead of local copy
 Also, build the swig wrapper from its source file.
--- a/setup.py
+++ b/setup.py
@@ -84,12 +84,11 @@
     PyWCSTools is distributed (and developed) as part of astLib.""",
     packages=['astLib', 'PyWCSTools'],
     package_data={'astLib': ['data/*']},
-    cmdclass={"build_ext": build_PyWCSTools_ext},
     scripts=exampleScripts,
     ext_modules=[
-        Extension('PyWCSTools._wcscon', [sourceDir+"wcscon_wrap.c"],
-        extra_objects=oFiles),
-        Extension('PyWCSTools._wcs', [sourceDir+"wcs_wrap.c"],
-        extra_objects=oFiles)
+        Extension('PyWCSTools._wcscon', [sourceDir+"wcscon.i"],
+        swig_opts=['-outdir', 'PyWCSTools/'], libraries=['wcstools']),
+        Extension('PyWCSTools._wcs', [sourceDir+"wcs.i"],
+        swig_opts=['-outdir', 'PyWCSTools/'], libraries=['wcstools'])
     ]
 	)
--- a/PyWCSTools/wcssubs-3.8.7/wcs.i
+++ b/PyWCSTools/wcssubs-3.8.7/wcs.i
@@ -14,7 +14,7 @@
 /* %feature("autodoc", "1") */
 
 %{
-#include "wcs.h"
+#include <wcs/wcs.h>
 %}
 
 /* WORKING */
